resources/debian-package/opt/jitsi/jibri/reload.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -X POST http://127.0.0.1:3333/jibri/api/internal/v1.0/notifyConfigChanged
```

Comment:
Can't we use SIGHUP instead? It's a more common approach to reloading the config of a running process.

Reply:
I think there is no general mechanism in java to listen for those ...

Reply:
` For the signals SIGINT, SIGTERM, and SIGHUP the Java Runtime is terminated by invoking the equivalent of System.exit with a value of 128 + the signal number. Other signals are ignored. `

Ouch. Ignore what I said...